libcontainer/system/syscall_linux_mipsx.go

```diff
@@ -0,0 +1,25 @@
+// +build linux,mips linux,mipsle linux,mips64 linux,mips64le
+
+package system
+
+import (
+	"syscall"
```

Comment:
I wonder if these are provided by `golang.org/x/sys/unix` (and so we could just switch `syscall_linux` to use that)?

Reply:
@vstefanovic Could you reply here? (otherwise PR looks good). Thanks!

Reply:
Unlikely, there's no Setuid and Setgid in golang.org/x/sys/unix/syscall_linux_*.go.

Reply:
Couldn't the implementation from `libcontainer/system/syscall_linux_64.go` have been used here? I'll sent a follow-up PR in #1613.